client/components/Editors/Table/index.tsx

```diff
@@ -25,8 +26,42 @@ export default function TableEditor(props: TableEditorProps) {
     () => createColumns(schema, report, history, selection),
     [schema, report, history, selection]
   )
+  const [rowsPerPage, setRowsPerPage] = React.useState(20)
+
+  const [rowHeight] = React.useState(40)
```

Comment:
@guergana why are you using `React.useState()` instead of the `useStore()` that we are already using through the application? Is there any difference?